1470.py

```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def shuffle(self, nums: List[int], n: int) -> List[int]:
+        res = []
+        pointer = n
```

Comment:
`pointer` is a redundant variable. In each iteration, you can always append elements to `res` by accessing indexes `i` and `i + n` for `i` from `0` to `n-1`.